frontends/bsd/src/app_root.tsx

```diff
@@ -566,6 +576,20 @@ export function AppRoot({
     );
   }
 
+  if (smartcard.status !== 'no_card' && !removeCardScreenShown) {
```

Comment:
Curious why we have to check `smartcard.status` here when `smartcard.status` is in a sense captured in `removeCardScreenShown`. Also curious why we display the card-removal card screen when `removeCardScreenShown` is false - I'd expect the opposite

Reply:
Again, there is a poor variable naming issue here.